app/views/patients/_new_patient.html.erb

```diff
@@ -8,10 +8,10 @@
   <%= bootstrap_form_for patient do |f| %>
     <div class="row">
       <div class="col-sm-2">
-        <%= f.text_field :primary_phone, label: 'Phone', value: phone %>
+        <%= f.text_field :primary_phone, label: 'Phone', value: phone, required: true, pattern: '[0-9]{10}' %>
```

Comment:
Yup! I didn't know html5 validators existed until recently. Anyways all three cases should pass.